### DISCUSSION TITLE
Show whats in the register using :reg, disappears in seconds

### DISCUSSION THREAD

**Discussion by iangregsondev** [LazyVim/LazyVim#4353] [Q&A] [answered]
Hi,

Quite new to lazyvim and wanted to show whats in the register, I am using the :reg 

And it does show me but in a message, and it disappears very quickly.

Am I am missing something ? Anyway to void this or an alternative command ?

Thanks in advance

Ian

**Comment by dpetka2001** [comment, marked as answer]
`:Noice all` or `<leader>sna` as a keybinding. For more info you should read `noice.nvim` github repo docs to get familiar with customizing it.

**Reply by iangregsondev** [reply]
Thank you.
